db/schema.sql

```diff
@@ -1,94 +1,104 @@
 -- We can use surah_number to make an api request to https://api.quran.com/api/v4/chapters/(surah_number)
 -- But because I'd prefer to store everything for ourselves, we should make the request to that api only once
 -- and store them here.
-CREATE TABLE Surah (
-    surah_number INTEGER PRIMARY KEY, -- This is the number corresponding to the surah in question
-    revelation_place VARCHAR(20),
-    revelation_order INTEGER,
+DROP TABLE IF EXISTS `Surah`; 
+CREATE TABLE IF NOT EXISTS Surah (
+    surah_id INTEGER NOT NULL, 
+    surah_number INTEGER NOT NULL,
+    revelation_place VARCHAR(100),
     name_complex VARCHAR(50),
     name_arabic VARCHAR(50), -- TODO: check if this is how we can store the arabic name, I think it's fine
-    verses_count INTEGER,
-    num_pages INTEGER,
-    english_name VARCHAR(50)
+    verse_count INTEGER NOT NULL,
+    PRIMARY KEY (surah_id)
 );
 
--- We've separated the notes because they could be used in multiple lessones
-CREATE TABLE Notes(
-    notes_id SERIAL PRIMARY KEY,
-    notes_file VARCHAR(200), -- This is the location of the notes file, essentially just the file name as we will be storing them all in the same place.
-    surah INTEGER,
-    version INTEGER, -- why is the word version highlighted in blue? This will be the version of the notes (i.e. v1.2). As the notes might get updated for the same surah
+DROP TABLE IF EXISTS `SurahInfo`; 
+CREATE TABLE IF NOT EXISTS SurahInfo(
+    title VARCHAR(255) NOT NULL,
+    info TEXT,
+    PRIMARY KEY (title)
+);
 
-    constraint SurahNotesFk foreign key (surah) references Surah(surah_number) on update cascade on delete cascade
+DROP TABLE IF EXISTS `Verse`; 
+CREATE TABLE IF NOT EXISTS Verse(
+    verse_index INTEGER NOT NULL,
+    surah INTEGER NOT NULL,
+    PRIMARY KEY (verse_index),
+    FOREIGN KEY (surah)
+        REFERENCES Surah(surah_id)
+        ON DELETE CASCADE
+        ON UPDATE CASCADE
 );
 
 -- This will store the locations for the video and audio file for a lesson
-CREATE TABLE Lesson (
-    lesson_id SERIAL PRIMARY KEY, -- This implies we will need to add the lesson number ourselves.
-    lesson_date DATE NOT NULL, 
-    source TEXT NOT NULL -- The host from which we will fetch the video (i.e. youtube or facebook)
+DROP TABLE IF EXISTS `Lesson`; 
+CREATE TABLE IF NOT EXISTS Lesson(
+    lesson_id INTEGER NOT NULL,  
```

Comment:
Whenever we are assigning IDs, it's best to use a serial value. See the line you deleted above. A serial value essentially updates to the next available integer automatically whenever inserting data. Once you fix this, the tests should work (after I've pushed my fixes to the tests)